### DISCUSSION TITLE
Is it possible to exceed the maximum speed limit (1110 deg/s) of the SPIKE Prime M motor?

### DISCUSSION THREAD

**Discussion by kubotafamily** [pybricks/support#1762] [Q&A] [answered]
Can this method, motor.control.limits(speed, acceleration, torque), exceed the maximum speed limit of the SPIKE Prime M motor (1110 deg/s)?
If this method cannot do this, please tell me what to do.

**Comment by laurensvalk** [comment, marked as answer]
The motor cannot go faster than that physically, so there isn't really a reason to increase it any further.

Why would you like to set it to an even higher value?

**Reply by kubotafamily** [reply]
To make a large walking robot move, I wanted the motor to rotate at high speed and with a large torque, but if that's physically impossible, I'll adjust the gear ratio to make it move.
I think it was possible with EV3's Pybricks...

**Reply by laurensvalk** [reply]
The EV3 motors were certainly a bit stronger, and could maintain the target speed even under a bit more load.
